docs/src/man/missing.md

```diff
@@ -54,9 +52,13 @@ julia> collect(skipmissing(x))
 
 ```
 
-`missing` elements can be replaced with other values via `Missings.replace`.
+The [Missings.jl](https://github.com/JuliaData/Missings.jl) package provides a few convenience functions to work with missing values.
+
+The function `Missings.replace` replaces `missing` elements with another value:

```

Comment:
Why do we recommend it over `coalesce.(x,1)` (I know it is lazy, but is it that useful?)

Reply:
Good idea. I now present `coalesce` first.